projects/packages/publicize/src/social-image-generator/utilities.php

```diff
@@ -21,6 +23,5 @@ function get_image_url( $post_id ) {
 		return '';
 	}
 
-	// TODO: update URL
-	return 'https://example.com/' . $token;
+	return add_query_arg( array( 'query' => rawurlencode( 't=' . $token ) ), Redirect::get_url( 'sigenerate' ) );
```

Comment:
As we don't use `site` we could pass change the Redirect::get_url call to:

`Redirect::get_url( 'sigenerate', array( 'site' => null ) )`